pkg/app/override.go

```diff
@@ -28,10 +29,13 @@ import (
 const (
 	// overrideKind is the override resource type.
 	overrideKind = "ksonnet.io/app-override"
-	// overrideVersion is the version of the override resource.
+	// OverrideVersion is the version of the override resource.
```

Comment:
The comment changed but the const did not. Was this intentional?

Reply:
No, there was a back and forth and I missed the comment when I changed it back. Thanks!